blog/en/blog/2022/08/28/fault-injection-testing-with-api-gateway.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Fault Injection Testing with API Gateway"
+authors:
+  - name: Bobur Umurzokov
+    title: Author
+    url: https://github.com/Boburmirzo
+    image_url: https://avatars.githubusercontent.com/u/14247607
+keywords: 
+- API gateway
+- Apache APISIX
+- Testing
+- Fault Injection
+- Microservices
+description: 
+The blog post describes how an **API Gateway** like [Apache APISIX](https://apisix.apache.org/) is useful for testing the robustness and resilience of microservices APIs. Throughout the post, we also get to know the types of possible failure injections with the **[Fault Injection Plugin](https://apisix.apache.org/docs/apisix/plugins/fault-injection/)** 🔌 and simulate failures on our existing [Product backend service](https://github.com/Boburmirzo/apisix-dotnet-docker/tree/main/ProductApi) (developed by using [ASP.NET Core WEB API](https://docs.microsoft.com/en-us/aspnet/core/?view=aspnetcore-6.0)).
```

Comment:
Hi @Boburmirzo, the description field will be injected to the HTML's description tag, please remove `**` or `[]()` 😊 Or it will be injected as plain text.

Reply:
Also, make sure the description is less than 150 words for better SEO.

Reply:
Great point!  I will remove that and make it short.